GLOSSARY.md

```diff
@@ -32,8 +32,8 @@ Examples for binders:
 ## Canonical Form
 A formula in canonical form has the property that its DeBruijn indices are
 minimized. For example when the formula `forall<0, 1> { 0: A && 1: B }` is
```

Comment:
Pre-existing, but I'm not sure this is how I would phrase it. It's not so much that the debruijn indices are *minimized*, but rather that free variables in the formula are assigned numbers starting from 0, based on the order in which they appear.

Also, this example is just not correct, or at least confusing to me. That is, this formula *is* in canonical form -- and, indeed, in closed form. But a formula like this `?3: A && ?4: B` is in neither -- the canonical form of it would be `?0: A && ?1: B`. 

I sometimes think of canonical form as converting into closed form -- that is, adding some binders at the outermost level, sort of like going from `?X: A && ?Y: B` to `exists<X, Y> { X: A && Y: B }`, but really that's not right. Canonicalization for example preserves the "universe" of each variable, and the solver acts a bit differently in that the variables that are enumerated in the canonicalization become part of the final solution that is returned.

Reply:
I see!